src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotificationForClaimantRepresented.java

```diff
@@ -74,14 +76,19 @@ private CallbackResponse notifyLipAfterNocApproval(CallbackParams callbackParams
         boolean isRespondentNotification = isRespondentNotification(caseEvent);
         boolean isApplicantSolicitorNotify = isApplicantSolicitorNotification(caseEvent);
         String templateId = getTemplateID(isRespondentNotification, isApplicantSolicitorNotify, caseData.isClaimantBilingual());
-        if (isNotEmpty(recipientEmail) && templateId != null) {
+        boolean eligibleForNotification = isNotEmpty(recipientEmail) && templateId != null;
+        if (eligibleForNotification) {
             notificationService.sendMail(
                 recipientEmail,
                 templateId,
                 isApplicantSolicitorNotify ? addPropertiesApplicantSolicitor(caseData) : addProperties(caseData),
                 String.format(REFERENCE_TEMPLATE_DEFENDANT, caseData.getLegacyCaseReference())
             );
         }
+
+        if (!eligibleForNotification) {
```

Comment:
why cant it be in just else

Reply:
code starts checking coverage - its just info i am printing so in case we find out if notification sent or not.